### PR TITLE
Incrby ms

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

File: src/tests/test_module.py [hunk 1]
```diff
@@ -783,8 +784,8 @@ def test_partial_madd(self):
             r.execute_command("ts.create", 'test_key3')
 
             now = int(time.time()*1000)
-            assert [now, 2000, 3000] == r.execute_command("ts.madd", 'test_key1', '*', 10, 'test_key2', 2000, 20, 'test_key3', 3000 , 30)
-            res = r.execute_command("ts.madd", 'test_key1', now + 1 , 10, 'test_key2', 1000 , 20, 'test_key3', 3001 , 30)
+            assert [now, 2000, 3000] == r.execute_command("ts.madd", 'test_key1', now, 10, 'test_key2', 2000, 20, 'test_key3', 3000, 30)
```
Comment (danni-m) [inline src/tests/test_module.py]: @gkorland the `*` also checks that `madd` supports the automatic timestamp variance

File: src/tests/test_module.py [hunk 2]
```diff
@@ -485,29 +485,30 @@ def test_incrby_reset(self):
...
-            time.sleep(1)
-            start_decr_time = int(time.time())
+            start_decr_time = int(time.time()*1000)
             for i in range(20):
                 r.execute_command('ts.decrby', 'tester', '1.5')
 
-            assert r.execute_command('TS.RANGE', 'tester', 0, int(time.time())) == [[start_incr_time, '100'], [start_decr_time, '70']]
+            result = r.execute_command('TS.RANGE', 'tester', 0, int(time.time()*1000))
+            assert result[-1][1] == '70'
+            assert len(result) <= 40
```
Comment (danni-m) [inline src/tests/test_module.py]: @gkorland any reference to check the values between `start_incr_time`, `start_decr_time`?

File: src/tests/test_module.py (hunk 1 above)
Comment (gkorland) [reply src/tests/test_module.py]: done

File: src/tests/test_module.py (hunk 2 above)
Comment (gkorland) [reply src/tests/test_module.py]: done

**Pull request merged**